content/commands/hugo.md

```diff
@@ -67,7 +67,6 @@ hugo [flags]
 
 ### SEE ALSO
 * [hugo benchmark](/commands/hugo_benchmark/)	 - Benchmark Hugo by building a site a number of times.
-* [hugo check](/commands/hugo_check/)	 - Contains some verification checks
```

Comment:
THis file is autogenerated and was correct.

Reply:
I committed what `hugo gen doc` generated.